test/common/deployment_types.go

```diff
@@ -259,3 +265,21 @@ func getDeployment() []Namespace {
 		},
 	}
 }
+
+func getAWSStorageDeployments() []Namespace {
```

Comment:
I think the name here is a bit confusing: they are in cluster databases, so they shouldn't be called  `AWSStorageDeployments`?

Reply:
You are right, is confusing. It was because these are the resources created in AWS when useClusterStorage is false. Changed now, thanks @wei-lee